packages/ember-simple-auth/node_modules/supports-color/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
why do we add `node_modules` folders to this repo?

Reply:
Ugh, added that accidentally.

This is still an early draft though – I'll let everyone know when this is ready for review.